install.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
 set -e # everything must succeed.
 if [ ! -d venv ]; then
-    virtualenv --python=`which python2` venv
+    virtualenv --python=`which python3.5` venv
```

Comment:
This install.sh script will use `python3.5` in a new virtualenv, but leave in place existing `python2` virtualenvs. Should either keep installing python2 by default and override it in some way if we want to support both; or check the current venv for the presence of `python3.5`.

Reply:
Also, the current dashboard stacks do not install python3: https://github.com/elifesciences/builder-private/blob/master/salt/top.sls#L25
e.g. Lax does it in this way: https://github.com/elifesciences/builder-private/blob/master/salt/top.sls#L96

Reply:
good catch, I overlooked that. this is how lax does it @seanwiseman : https://github.com/elifesciences/lax/blob/develop/install.sh#L7-L15

Reply:
Thank you, both.

I have just replicated what you do in lax @lsh-0 . I will put the builder-private PR together now.